app/controllers/concerns/api/import_puppetclasses_common_controller.rb

```diff
@@ -65,7 +65,7 @@ def changed_environments
         end
       end
     rescue => e
-      if e.message =~ /puppet feature/i
+      if /puppet feature/i.match?(e.message)
```

Comment:
This feels like it deserves a dedicated exception class instead of parsing error messages.